src/components/preview/index.js

```diff
@@ -28,9 +41,43 @@ export const Preview = ({ title, description, mimeType, previewUri, tags }) => {
         })}
       </div>
       <div className={styles.info}>
-        <div className={styles.title}>{title}</div>
-        <div className={styles.description}>{description}</div>
-        <Tags token_tags={token_tags} preview={true} />
+        <div className={styles.title}>
+          <strong>Title:</strong>
+          {title}
+        </div>
+        <div className={styles.description}>
+          <strong>Description:</strong>
+          {description}
+        </div>
+
+        <div className={styles.description}>
```

Comment:
maybe rename `styles.description` to something more general, like `styles.field` (if it's not just used for the description)